src/_guides/language/analysis-options.md

```diff
@@ -121,15 +127,10 @@ analyzer:
 
 You can use the flags together or separately;
 both default to `true`.
-The presence of either flag, regardless of value, enables
-the Dart 2 type system.
 
-{% comment %}
-**PENDING:
-Will these flags still appear under strong-mode in Dart 2.0?
+**[PENDING: Do these flags still appear in Dart 2.0/2.1?
 Should we mention related command-line flags
-(--no-implicit-casts, --no-implicit-dynamic)?**
-{% endcomment %}
+(--no-implicit-casts, --no-implicit-dynamic)?]**
```

Comment:
The flags are still available, but in general I think users are better off using the analysis options file because they'll get the same behavior from both the command-line and any server-based tooling they use, and everyone who works on the package will get the same behavior as well.

Reply:
Makes sense. Thanks, @bwilkerson!